src/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+import arrowmenu from './../../images/arrowmenu.svg';
+import arrowSecond from './../../images/arrow-second.svg';
+
+const Accordion = ({ title, text }) => {
+  const [selected, toggle] = useState(false);
+  return (
+    <div className='accordion'>
+      <div className='title'>
+        <p> {title} </p>
+        <span onClick={() => toggle(!selected)}>
+          {selected ? (
+            <img src={arrowSecond} alt='' />
+          ) : (
```

Comment:
When I open and close accordion menu, background in header changes its position

https://user-images.githubusercontent.com/85338076/215133288-a9f6efee-0c9a-4100-b63c-4687fe9e8fbe.mov